wireapp/wire-android-reloaded
# feat(calling): Display caller name

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

When we receive a group call, we don't see who is calling

### Solutions

Show the caller name in incoming call screen when receiving a group call

Needs releases with:


### Testing

#### How to Test

Receive a group call
you should see "someone is calling" in the incoming call screen

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
